metadata-service/factories/src/main/resources/application.yml

```diff
@@ -24,7 +24,13 @@ authentication:
   # The max duration of a UI session in milliseconds. Defaults to 1 day.
   sessionTokenDurationMs: ${SESSION_TOKEN_DURATION_MS:86400000}
 
-
+datahub:
+  gms:
+    host: ${DATAHUB_GMS_HOST:${GMS_HOST:localhost}}
+    port: ${DATAHUB_GMS_PORT:${GMS_PORT:8080}}
+    useSSL: ${DATAHUB_GMS_USE_SSL:false}
+    sslContext:
+      protocol: ${DATAHUB_GMS_SSL_PROTOCOL:#{null}}
```

Comment:
Guessing these don't need to change?

Reply:
Oh thought these were added by your PR. Adding these as well